packages/angular_devkit/build_angular/src/builders/application/index.ts

```diff
@@ -42,6 +42,23 @@ export async function* buildApplicationInternal(
   }
 
   const normalizedOptions = await normalizeOptions(context, projectName, options);
+
+  // Warn about prerender/ssr not yet supporting localize
+  if (
```

Comment:
Are you planning to get back to this into a separate PR? SSG and AppShell support i18n is a fully supported feature right now with the webpack based builders.

Reply:
Yes in a followup.  app shell support needs some additional refactoring and the E2E test needs a fairly substantial rework.